docs/unit-configuration.md

```diff
@@ -104,9 +104,18 @@ These properties are configurable at the unit, investigation, or problem levels
 ```typescript
   disabledFeatures: string[];
   toolbar: SnapshotIn<typeof ToolbarModel>;
+  authorTools: SnapshotIn<typeof ToolbarModel>;
   // required tools that aren't in the toolbar can be specified here
   tools?: string[];
   defaultDocumentTemplate?: IAuthoredDocumentContent;
+  planningTemplate?: Record<string, IAuthoredDocumentContent>;
```

Comment:
Was `planningTemplate` meant to be added here?

Reply:
Yes, trying to keep the documentation up to date with `problem-configuration.ts`.